src/lib/components/Header.svelte

```diff
@@ -3,12 +3,18 @@
 	import Logo from '$components/Logo.svelte'
 	import { page } from '$app/stores'
 	import Button from '$components/Button.svelte'
+	import Banner from '$components/Banner.svelte'
 
 	$: onHomepage = $page.url.pathname == '/'
 
 	let open = false
 </script>
 
+<Banner
+	message="Participez à notre conférence de lancement du 10 septembre <a href=&quot;https://pauseia.fr/propositions&quot; target=&quot;_blank&quot;>en vous inscrivant ici</a>"
+	bgColor="var(--brand)"
+/>
+
```

Comment:
Once you use `<slot>` you can put the message as a child. You should also use the conference link.
```suggestion
<Banner>
Participez à notre conférence de lancement du 10 septembre <a href="https://lu.ma/t648gs2x">en vous inscrivant ici</a>
<Banner/>
```

Reply:
done